### PR TITLE
Relicence

### PR DESCRIPTION
**Pull request by seporaitis**

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,12 @@
 History
 -------
 
+2.0.3 (2016-09-13)
```
Comment (adamchainz) [inline HISTORY.rst:6]: 2.0.2 surely?

File: setup.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
 setup(
     name='poor-smime-sign',
-    version='2.0.2',
```
Comment (adamchainz) [inline setup.py:24]: oh i see, why was there no changelog for 2.0.2 ?

Comment (seporaitis) [reply setup.py:24]: Correct :)

**Comment by adamchainz** [comment]
@seporaitis bump

**Pull request merged**